Basic/Bad/bad.pd

```diff
@@ -1390,20 +1390,20 @@ Can be done inplace.
 
 =for example
 
- $a = byte( [0,1,3] );
+ $x = byte( [0,1,3] );
  $mask = byte( [0,0,0] );
  set($mask,1,$mask->badvalue);
- $a->inplace->copybad( $mask );
- p $a;
+ $x->inplace->copybad( $mask );
+ p $x;
```

Comment:
This example doesn't seem to work but it is not due to the variable name changes.  There needs to be a `$mask->badflag(1)` or `badflag($mask,1)` before the `set` function.